src/graphnet/models/graphs/graphs.py

```diff
@@ -100,3 +108,203 @@ def __init__(
             seed=seed,
             **kwargs,
         )
+
+
+class KNNGraphRRWP(GraphDefinition):
+    """KNN Graph with relative random walk probabilities."""
```

Comment:
Similarly to the other representation, I would suggest expanding the doc string so it's easier to see how this differs from existing representations and how the new fields are accessible. I think there are five new fields in the RRWP case. So here's an example:

"""
A KNN graph representation with Relative Random Walk Probabilities (RRWP).

Identical to KNNGraph but with five additional fields:  

`abs_pe = graph['abs_pe']` # Absolute positional encoding
`rrwp_index = graph['rrwp_index']` # rrwp index (which is used for slicing the vals?)
`rrwp_val = graph['rrwp_val']` # rrwp values
`degree= graph['deg']` # Degree of each node (num. of incoming edges)
`log_degree = graph['log_deg']` # Equal to torch.log10(graph['deg'] + 1)
"""